engineio/src/client/client.rs

```diff
@@ -1,662 +0,0 @@
-use super::super::socket::Socket as InnerSocket;
```

Comment:
Yes, it was to fix the lint, I figured since the lint was enabled that it just hadn't got around to being fixed yet, I can put the code back, and disable the lint.